src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -402,6 +376,7 @@ protected void run() {
         registerCommand(new UpdateJDACommand());
         registerCommand(new ChangelogCommand());
         registerCommand(new JumboCommand());
+        registerCommand(new ColourCommand());
```

Comment:
Ok we should probably deside on how we want to hanlde this in code. you know what I'm talking about